examples/make_eeg_bids.py

```diff
@@ -0,0 +1,177 @@
+"""
```

Comment:
I get:

```py
SyntaxError: Non-ASCII character '\xe2' in file /home/mainak/Desktop/projects/github_repos/mne-bids/examples/make_eeg_bids.py on line 145, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

when I tried to run this example

Reply:
How did you run it? I am using the mne-bids environment installed via conda and the mne-python and mne-bids `environment.yml` ... and then running `python mne_bids/examples/make_eeg_bids.py` gives me no errors ... (python 3 everything)

Reply:
but this is probably due to the `├──` pip characters I am using ... I thought they'd be pretty to show the directory structure. Generated with the `tree` bash function

Reply:
I used python 2.7 :-) That's why probably ...

Reply:
mmmh as long as it's there, I might as well make a small effort for python 2 ;-) I replaced these with slightly more ugly dashes now that should not throw encoding errors anymore

Reply:
haha, my default env is still 2.7 for some reason. I should switch soon :)

Reply:
@jasmainak, wait, aren't we ending support for py2? ;) https://github.com/mne-tools/mne-python/pull/5256